src/test/java/seedu/findvisor/model/person/NameTest.java

```diff
@@ -35,7 +35,9 @@ public void isValidName() {
         assertTrue(Name.isValidName("12345")); // numbers only
         assertTrue(Name.isValidName("peter the 2nd")); // alphanumeric characters
         assertTrue(Name.isValidName("Capital Tan")); // with capital letters
+        assertTrue(Name.isValidName("John                           Doe")); // with multiple spaces
         assertTrue(Name.isValidName("David Roger Jackson Ray Jr 2nd")); // long names
+        assertTrue(Name.isValidName("David Roger  Jackson Ray  Jr  2nd")); // long names with multiple spaces
```

Comment:
Similarly for these two newly added lines, perhaps would be better to shift these cases to `ParserUtilTest.java` to assert their equality when parsing names with multiple spaces

Reply:
I agree with these comments, will work to correct the testing.